### DISCUSSION TITLE
Automatic Updating Contributor List on Readme

### DISCUSSION THREAD

**Discussion by Ayushjhawar8** [Difegue/LANraragi#1103] [Ideas]
**Please describe your suggestion, and the problem it'd solve.**
I’m excited to propose the implementation of an automated contributor list in our README file. Currently, we manually update this list, which can be tedious and often leads to outdated or incomplete information about our contributors. Automating this process will ensure that everyone who contributes is recognized promptly and accurately.

**Additional context**
By integrating the All Contributors bot, we can automatically update the contributors list whenever someone makes a contribution. This feature will enhance our project documentation, improve contributor recognition, and reduce the manual effort required to keep the README up to date. It’s a great way to encourage participation and foster a sense of community within our project.

**Comment by Ayushjhawar8** [comment]
Sample Screenshot:
![image](https://github.com/user-attachments/assets/508bb280-c816-4daa-9572-e9f262ae3d5c)

Sample Screenshot:
![image](https://github.com/user-attachments/assets/c6cb01f6-853e-4e5a-8b0e-d4741a4072e4)

**Comment by Ayushjhawar8** [comment]
if liked the changes please assign me

**Comment by Difegue** [comment]
Nope sorry, don't really see the point of adding a workflow for this when GH already does it on the side of the repo:  
![image](https://github.com/user-attachments/assets/821cd47b-1a73-46a6-bbfa-f84ed1cef362)
